index.html

```diff
@@ -45,5 +45,194 @@ <h1 class="title">I'm Mariane <br>Glad to see you!</h1>
+            <img src="images/Snapshoot Portfolio3.svg" alt="" class="tonic-1">
+            <div>
+                <h1 class="header-tonic">Tonic</h1>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2015</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+
+        <div class="works works2">
+            <img src="images/Snapshoot Portfolio (3).svg" alt="" class="tonic">
+            <img src="images/Snapshoot Portfolio4.svg" alt="" class="tonic-1">
+            <div>
+                <h1 class="stories">Multi-Post Stories</h1>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2015</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-myself" id="about-myself">
+        <div class="myself">
+            <div>
+                <h1 class="about">About Myself</h1>
+                <p class="experience">Hello I'm a software developer! 
+                        I can help you build a product , feature or website Look 
+                        through some of my work and experience! If you like what you 
+                        see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <p class="connect-1">LET'S CONNECT</p>
+
+                <ul class="icons-1">
```

Comment:
- [ ] Since` LET'S CONNECT` is a title/heading for the social icons, I would suggest using `heading` tag instead of `p` tag.  

_Note: keep in mind the hierarchy of these tags e.g.` h3` can't come before` h2 `and `h3 ` can't be placed after` h4` etc._

Reply:
Done